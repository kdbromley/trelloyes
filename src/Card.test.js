const { findAllInRenderedTree } = require("react-dom/test-utils");
import renderer from 'react-test-renderer';
import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';


describe('Card component', () => {
    it('renders w/o crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders UI as expected', () => {
        const tree = renderer 
        .create(<Card title='First Card' content='lorem ipsum' />)
        .toJSON();
        expect(tree).toMatchSnapshot();
    })
})