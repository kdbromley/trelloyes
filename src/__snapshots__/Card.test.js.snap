// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders UI as expected 1`] = `
<div
  className="card"
>
  <h3>
    First Card
  </h3>
  <p>
    lorem ipsum
  </p>
</div>
`;
